pkgs/development/compilers/ciao/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "ciao";
+  version = "1.19.0";
+  src = fetchFromGitHub {
+    owner = "ciao-lang";
+    repo = "ciao";
+    rev = "v${version}";
+    sha256 = "03qzcb4ivgkiwdpw7a94dn74xqyxjwz5ilrr53rcblsh5ng299jp";
+  };
+
+  buildInputs = [ ];
```

Comment:
Why is this necessary? I'd expect `buildInputs` is empty by default? :thinking:

Reply:
In the future there may be `readline` dependency. So I'd like to leave it as is.

Reply:
If you'd like to leave info for future packaging, a comment would be the appropriate way. That doesn't really feel warranted, though. I'd prefer consistency with other packages.